Fix typo for Cryogenic Treatment nanite program

CMOisLing opened the pull request:
The nanites rapidly (skin -> sink) heat through the host's skin, lowering their temperature.

Alexandria404 commented:
I don´t speak coder, please translate.

nichlas0010 commented:
wen admen leaf srvr, srvr sent msg bot.

bot no do thing yet, but me mak dat suun

Alexandria404 commented:
And it alarms admin chat then I assume? Nice.

nichlas0010 commented:
nut yet, dat part is in yogbutt. butt wen dis b merge, i mak yogbutt bit.

CMOisLing committed:
Merge pull request #9 from yogstation13/master

upodate

CMOisLing committed:
Merge pull request #10 from yogstation13/master

f

CMOisLing committed:
Merge pull request #11 from yogstation13/master

ffff

CMOisLing committed:
skin -> sink

The nanites rapidly sink* heat through the host's skin, lowering their temperature.
